plugins/Wallet/js/components/wallet.js

```diff
@@ -1,31 +1,37 @@
 import React, { PropTypes } from 'react'
 import TransactionList from '../containers/transactionlist.js'
-import SendButton from '../containers/sendbutton.js'
+import SendButton from './sendbutton.js'
 import SendPrompt from '../containers/sendprompt.js'
 import ReceiveButton from '../containers/receivebutton.js'
 import ReceivePrompt from '../containers/receiveprompt.js'
 import NewWalletDialog from '../containers/newwalletdialog.js'
 
-const Wallet = ({confirmedbalance, unconfirmedbalance, showReceivePrompt, showSendPrompt, showNewWalletDialog }) => (
-	<div className="wallet">
-		<div className="wallet-toolbar">
-			<div className="balance-info">
-				<span>Confirmed Balance: {confirmedbalance} SC </span>
-				<span>Unconfirmed Delta: {unconfirmedbalance} SC </span>
+const Wallet = ({confirmedbalance, unconfirmedbalance, siafundbalance, showReceivePrompt, showSendPrompt, showNewWalletDialog, actions }) => {
+	const onSendClick = (currencytype) => () => actions.startSendPrompt(currencytype)
+	return (
+		<div className="wallet">
+			<div className="wallet-toolbar">
+				<div className="balance-info">
+					<span>Confirmed Balance: {confirmedbalance} SC </span>
+					<span>Unconfirmed Delta: {unconfirmedbalance} SC </span>
+					{siafundbalance !== '0' ? (<span> Siafund Balance: {siafundbalance} SF </span>) : null}
+				</div>
+				{siafundbalance !== '0' ? <SendButton currencytype="siafunds" onClick={onSendClick('siafunds')} />: null}
```

Comment:
fyi, I think `if x { <foo></foo> }` is equivalent to `x ? <foo></foo> : null`. Might be cleaner in some cases.